lib/evss/pciu_address/address.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require 'common/models/base'
+
+module EVSS
+  module PCIUAddress
+    class Address < Common::Base
```

Comment:
Seems like the only thing this needs from `< Common::Base` is 
```
include ActiveModel::Serialization
```
and, 
```
include Virtus.model(nullify_blank: true)
```
Otherwise, `PCIUAddress` does not really have an "`is a`" relationship with `Common::Base` which seems to be models that need sorting and pagination.  It probably doesn't need to change here, I'm more just ranting because outside of `Common::RedisStore`, our base model classes don't seem so "common".